features/facebookImport/src/model/entities/facebook-account.js

```diff
@@ -235,6 +236,14 @@ class FacebookAccount {
         return this._relatedAccounts.count;
     }
 
+    get comments() {
+        return this._comments;
+    }
+
+    set comments(comments) {
```

Comment:
Again, do we need this setter?

Reply:
In this case yes. The importer is a subclass of `DirectKeyDataImporter` that uses the setter.